ui/src/app/machines/views/MachineDetails/ActionConfirm/ActionConfirm.tsx

```diff
@@ -22,7 +22,7 @@ type Props = {
   onConfirm: () => void;
   onSaveAnalytics: AnalyticsEvent;
   statusKey: keyof MachineStatus;
-  submitAppearance?: "negative" | "positive";
+  submitAppearance?: "negative" | "neutral" | "positive";
```

Comment:
Maybe we should just change this to `ButtonProps["appearance"]`?

Reply:
Done.